docs/_docs/Develop/script-js.md

```diff
@@ -247,6 +254,8 @@ Specific wine architecture ("x86" or "amd64"):
 ```
 Specific windows version:
 ```javascript
+include(["engines", "wine", "plugins", "windows_version"]);
+        ...
```

Comment:
Why the deep indention?

Reply:
It is the same than in SteamScript.

Reply:
@plata what do you think? Should we always indent the `...` or should we not indent the `...` at all?

Reply:
Can you show a comparing screenshot?